docroot/themes/contrib/civictheme/includes/media.inc

```diff
@@ -59,6 +61,9 @@ function civictheme_preprocess_media__civictheme_video(&$variables) {
 
 /**
  * Implements hook_preprocess__HOOK().
+ *
+ * @SuppressWarnings(PHPMD.StaticAccess)
+ * @SuppressWarnings(PHPMD.MissingImport)
```

Comment:
This function still throwing warnings for the excluded rules
@AlexSkrypnyk

Reply:
Hacked around it - seems to be an issue with assigning variables to returns from static methods.